src/tool_getparam.c

```diff
@@ -1027,6 +1029,49 @@ static const struct LongShort *single(char letter)
   return singles[letter - ' '];
 }
 
+struct TOSEntry {
+  const char *name;
+  unsigned char value;
+};
+
+static const struct TOSEntry tos_entries[]= {
+  {"AF11", 0x28},
+  {"AF12", 0x30},
+  {"AF13", 0x38},
+  {"AF21", 0x48},
+  {"AF22", 0x50},
+  {"AF23", 0x58},
+  {"AF31", 0x68},
+  {"AF32", 0x70},
+  {"AF33", 0x78},
+  {"AF41", 0x88},
+  {"AF42", 0x90},
+  {"AF43", 0x98},
+  {"CS0", 0x00},
+  {"CS1", 0x20},
+  {"CS2", 0x40},
+  {"CS3", 0x60},
+  {"CS4", 0x80},
+  {"CS5", 0xa0},
+  {"CS6", 0xc0},
+  {"CS7", 0xe0},
+  {"ECT0", 0x02},
```

Comment:
Are you sure it even makes sense to allow ECT(0) and ECT(1) values?  Linux masks these bits out for SOCK_STREAM sockets anyway.

https://github.com/torvalds/linux/blob/29c73fc794c83505066ee6db893b2a83ac5fac63/net/ipv4/ip_sockglue.c#L590-L592